tasks/update_appraisal_gemfiles.rake

```diff
@@ -0,0 +1,13 @@
+desc 'update gemfiles/*.gemfile and gemfiles/*.lock files based on Appraisals'
```

Comment:
💯

Reply:
errr, i found this to be not super consistent if there are existing gemfile.lock files... do you think we should `rm -r gemfiles/.` to start? or is that overkill

also what about `bundle install` to start before we do any of the docker stuff?

Reply:
We can try `bundle exec appraisal update` instead. I believe that's what I normally use to update dependencies.